src/test/suite/fixtures/recorded/languages/rust/changeNameDrum3.yml

```diff
@@ -0,0 +1,36 @@
+languageId: rust
```

Comment:
I _think_ I'd also support "key" on this one, just because of the way it looks, though tbh I'm not totally sure what this code actually does 😅

Reply:
Given key-value literals aren't really a thing in Rust I'd be okay with letting `key` match struct field names.

Reply:
> Hmm, on the note of one of the above threads, this'd be an interesting test:
> 
> ```rust
> if let User { first_name: name, surname: "Doe" } = user {
> }
> ```
> 
> I'd expect the scopes when targeting within `first_name: name` to behave the same way as destructuring in JavaScript.

Which is how exactly?